resultsdbupdater/utils.py

```diff
@@ -88,9 +88,10 @@ def post_to_resultsdb(msg):
             msg['body']['msg']['team'], msg['body']['msg']['job_names'])
     else:
         LOGGER.warn((
-            'The message "{0}" did not contain a team. Using just "job_names" '
-            'as the name for the Test Case').format(msg_id))
-        testcase_name = msg['body']['msg']['job_names']
+            'The message "{0}" did not contain a team. Using '
+            '"unassinged\job_names" as the name for the Test Case')
+                .format(msg_id))
+        testcase_name = 'unassigned\{0}'.format(msg['body']['msg']['job_names'])
```

Comment:
Yeah, all I can find is that the Fedora instance uses dots to delimit their testcase names with a namespace, and I'd prefer that we do the same:  https://taskotron.fedoraproject.org/resultsdb_api//api/v1.0/testcases/dist.depcheck/results